app/views/coronavirus_form/which_goods.html.erb

```diff
@@ -0,0 +1,73 @@
+<% content_for :title do %><%= t('coronavirus_form.which_goods.title') %><% end %>
+<% content_for :extra_headers do %>
+  <meta name="description" content="<%= t('coronavirus_form.which_goods.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<main id="main-content" role="main">
```

Comment:
We shouldn't need these wrappers here as we already have them in the layout. We can start straight with the validation_error.

Reply:
Awesome! Thanks